Core/MIPS/ARM/ArmJit.cpp

```diff
@@ -634,8 +646,14 @@ void ArmJit::Comp_Generic(MIPSOpcode op)
 	if ((info & IS_VFPU) != 0 && (info & VFPU_NO_PREFIX) == 0)
 	{
 		// If it does eat them, it'll happen in MIPSCompileOp().
-		if ((info & OUT_EAT_PREFIX) == 0)
+		if ((info & OUT_EAT_PREFIX) == 0) {
 			js.PrefixUnknown();
+
+			// Even if DISABLE'd, we want to set this flag so we overwrite.
+			const char *name = MIPSGetName(op);
+			if (!strcmp(name, "mtv") || !strcmp(name, "vmtvc"))
+				js.blockWrotePrefixes = true;
```

Comment:
Let's use an op flag fetched with MIPSGetInfo(op) instead of doing string compares, maybe? We already have plenty of flags in MIPSTables.h like VFPU_NO_PREFIX etc.

Reply:
Ah, I thought we were out of bits but I forgot we went to 48.

-[Unknown]

Reply:
Nice, thanks!